evm_loader/tests/test_transaction.py

```diff
@@ -449,5 +450,84 @@ def test_07_combined_continue_gets_before_the_creation_of_accounts(self):
                                                                           step=Step.Complete)
 
 
+# def test_fail_on_no_signature(self):
```

Comment:
Do we need this commented test?

Reply:
No, in develop branch they already removed